stable/prometheus-adapter/templates/hpa-metrics-cluster-role-binding.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and .Values.rbac.create .Values.rules.resourceRules -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
Filename should include resource like `resource-metrics-cluster-role-binding.yaml`. I don't think it should have HPA in the name as it's not related to HPA.